drivers/sensor/mma8451q/Kconfig

```diff
@@ -0,0 +1,41 @@
+# Kconfig - MMA8451Q Accelerometer sensor configuration options
+
+#
+# Copyright (c) 2018 Lars Knudsen
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig MMA8451Q
+	bool
+	prompt "MMA8451Q Accelerometer"
+	depends on I2C
+	help
+	  Enable driver for MMA8451Q Accelerometer.
+
+config MMA8451Q_NAME
```

Comment:
Please get the driver name, I2C address and I2C dev name from dts. See `drivers/sensor/fxos8700` and `dts/bindings/sensor/nxp,fxos8700.yaml` for an example.

Reply:
This took me a while to figure out - but I think I got it right by adding an entry for the mma8451q in the i2c section of the frdm-kl25z board def + a dts.fixup file.